rfcs/20210510-binary-size-reduction/README.md

```diff
@@ -0,0 +1,156 @@
+# oneDNN binary size reduction through build system.
+
+## Problem Statement.
+Several oneDNN users due to their usage model specifics would love to see their
+application linked statically with oneDNN (or shared object of the library
```

Comment:
Instead of "to see their application...doesn't exceed..." --> "to make sure their application..."

Instead of "love" --> "very much like"?

Reply:
Hi @jedippold, thanks a lot for your comments. This one is particular difficult for me to process. That's why I re-wrote it completely. Could you take a second look on it, please?
I resolved and addressed rest you've left, thanks for suggestions.